pkg/apis/migration/v1alpha1/directvolumemigrationprogress_types.go

```diff
@@ -64,11 +63,9 @@ type DirectVolumeMigrationProgress struct {
 }
 
 func (d *DirectVolumeMigrationProgress) MarkReconciled() {
-	u, _ := uuid.NewUUID()
 	if d.Annotations == nil {
 		d.Annotations = map[string]string{}
 	}
-	d.Annotations[TouchAnnotation] = u.String()
```

Comment:
This TouchAnnotation is an unintended leftover from the remote watch system. This was likely causing requeues where they shouldn't have been happening in DVM/DVMP.

Reply:
+1000, thanks for removing this